packages/cookie_store/lib/cookie_store.dart

```diff
@@ -0,0 +1,18 @@
+/// An [RFC6265](https://httpwg.org/specs/rfc6265.html) compliant cookie store.
+///
+/// Cookies are stored inside of `CookieStore`s parsing the received cookies
+/// into `StorableCookie`s that can be saved in a `CookiePersistence`.
+///
+/// This package has an implementation independent api and provides utilities
+/// that make it easy to extend and implemented into existing storage backends
+/// like SQLite databases.
+///
+/// Users of the [cookie_jar](https://pub.dev/packages/cookie_jar) package
+/// can use the provided `CookieJarAdapter`.
```

Comment:
I don't think a separate package makes sense just for an adapter class.
If you really want to stop depending on cookie_jar we should require it in the nc package and publish the cookie store.

Reply:
Why not remove cookie jar entirely, even from the nextcloud package? I think it is fine if everyone just uses our cookie store instead of the cookie jar.